src/styles/global.css

```
@@ -35,6 +35,9 @@ html {
   --breakpoint-mobile: 767px;
   --breakpoint-tablet: 640px;
   --breakpoint-large: 1024px;
+
+  /* max widths */
+  --max-content-width: 1200px;
```

there's one other with the value 1400 and one for 1000 I believe. We can add and adjust for the others in a separate PR, but what about calling it e.g.:

```suggestion
  --max-content-width-large: 1400px;
  --max-content-width-medium: 1200px;
  --max-content-width-small: 1000px;
```